base_layer/core/src/transactions/transaction/transaction_input.rs

```diff
@@ -56,11 +58,29 @@ use crate::{
     },
 };
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
+#[repr(u8)]
+pub enum TransactionInputVersion {
+    DIBBLER = 0,
+}
+
```

Comment:
I think we need a `TransactionInputVersion::GetCurrentVersion()` implementation here.